semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -169,14 +177,14 @@ def to_markdown_attr(
 
     def derive_requirement_level(self, attribute: SemanticAttribute):
         if attribute.requirement_level == RequirementLevel.REQUIRED:
-            required = "Required"
+            required = "`Required`"
```

Comment:
Why use code formatting here? It's not code(-like) IMHO.

Reply:
The motivation is provided here: https://github.com/open-telemetry/semantic-conventions/pull/283#discussion_r1361413434

people not deeply familiar with otel assume that the requirement level is plain English. Having formatting emphasizes it.